packages/manager/apps/zimbra/src/components/layout-helpers/Dashboard/Dashboard.tsx

```diff
@@ -76,6 +76,12 @@ export const Dashboard: React.FC = () => {
       to: `${basePath}/email_accounts`,
       pathMatchers: computePathMatchers([urls.email_accounts]),
     },
+    {
+      name: 'mailing_list',
+      title: t('zimbra_dashboard_mailing_list'),
+      to: `${basePath}/mailing_list`,
```

Comment:
```suggestion
      to: `${basePath}/mailing_lists`,
```
The rest of url of tabs it's in plurial